### PR TITLE
Blog6 styles

### PR DESCRIPTION
**Pull request by dpaquette**

### CODE REVIEW

File: Blog 6 - Application Style and Layout.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
+#Application Style and Layout
+We have come a long way in our process of updating Bug Tracker.NET. We have made major improvements to the code structure, reducing duplication, improving security, and paving the way for using newer technology. As software developers, it is easy for us to look at the changes and see the value. As a user, these changes have largely been invisible. The application has changed very little from the user's perspective. It is finally time to make some changes to the UI.
+
+Simon and I are both developers, not designers. You might think that we are not necessarily the best suited to tackle a redesign of the UI. Ideally, this work would be completed with an experienced designer on the team. The reality is that we often do not have access to designers, especially when working on internal line of business applications. Despite our lack of formal design training, Simon and I have both had some great success with redesigning  older applications. This is one area where the impact can be much bigger than you might expect.
```
Comment (stimms) [inline Blog 6 - Application Style and Layout.md]: Hey, I went to dribble twice last week that makes me Almost a Designer™

File: Blog 6 - Application Style and Layout.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+Here is a little anecdote from Simon:
```
Comment (stimms) [inline Blog 6 - Application Style and Layout.md]: You can just claim this for your own. No need to mention me.

File: Blog 6 - Application Style and Layout.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+This installs a set of CSS, JavaScript and font files into the application Content and Scripts folders. Now we need to ensure that each page in BugTracker.NET properly references these files. In previous versions of BugTracker.NET, this would have been a daunting task. Luckily, we recently introduced master pages. Now, we can do easily add these references everywhere by adding the references to the 2 master pages.
+
+In the head tag, we add a reference to the main CSS file:
+
+`<link href="Content/bootstrap.min.css" rel="stylesheet" />`
+
+Also in the head tab, we will add a couple meta tags. The first one is to ensure the latest rendering engine is used in Internet Explorer. The second is to ensure the page width resizes properly on mobile browsers:
+
+````
+     <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (stimms) [inline Blog 6 - Application Style and Layout.md]: Oh, good idea

File: Blog 6 - Application Style and Layout.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+
+As an added bonus, the responsive Bootstrap layout also looks good on a smaller screens.
+
+![Create Bug on small screens](Images/CreateBugSmall.png)
+
+[View the commit - Updated Create Bug form to use Bootstrap layout](https://github.com/dpaquette/BugTracker.NET/commit/a74922469148d02f1c4d7166bf4114ca983ca74f)
+
+Next, we will tackle the Edit Bug screen. It turns out this is the same page as the Create Bug page, but with some sections displayed instead of hidden.
+
+![The original edit bug form](Images/EditBugOriginal.png)
```
Comment (stimms) [inline Blog 6 - Application Style and Layout.md]: Might specifically mention that this is before the main menu changes

File: Blog 6 - Application Style and Layout.md [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+
+We can use the grid layout from Bootstrap to position the buttons on the left side of the form. After applying some standard button styling and moving a few things around, we end up with a relatively modern and attractive looking Edit Bug form.
+
+![Edit Bug Form on a large screen](Images/EditBugLarge.png)
+
+[View the commit - Updated Edit Bug Page](https://github.com/dpaquette/BugTracker.NET/commit/3376407a50e41737c32ae8623743dff99d88d0f4)
+
+While the layout we created here looks fine on a desktop or tablet, things get a little strange on a phone-sized display. The buttons on the left hand side don't really fit on a smaller screen. It sure would be nice if we could collapse this into a dropdown menu when displayed on a narrow screen.
+
+By applying the same approach as we did to the main menu, we are able to collapse the bug toolbar on smaller screens and show the Bug Tools dropdown button instead. It's moments like this that make me glad that frameworks like Bootstrap exist.
```
Comment (stimms) [inline Blog 6 - Application Style and Layout.md]: This is cool, I didn't realize you could do this.

File: Blog 6 - Application Style and Layout.md [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+
+![Edit Bug Form on small screen](Images/EditBugSmall.png)
+
+[View the commit - Collapse Edit Bug tools on smaller screen](https://github.com/dpaquette/BugTracker.NET/commit/9297f22e18719d307bc7f4c433b25fdd100592aa)
+
+We could continue to make changes to more pages, but this is probably a good time to publish the update to the application. The changes we made cover the most commonly used pages. Now is a good time to get some feedback from users before applying the styling changes to the remaining application pages.
+
+##Conclusion
+With very little effort, we were able to make some high impact improvemnts to the design of a web application that is over a decade old. By using a popular CSS framework, we were able to leverage design expertise from the community without requiring extensive design experience ourselves. As an added bonus, we are  also able to provide a much more pleasant experience for those accessing BugTracker on a mobile devices.
```
Comment (stimms) [inline Blog 6 - Application Style and Layout.md]: Maybe put in a plug for James' book. Gotta pay the butcher bill*.


*I don't know what that means.

File: Blog 6 - Application Style and Layout.md (hunk 4 above)
Comment (dpaquette) [inline Blog 6 - Application Style and Layout.md]: I think I will just crop the image and remove the header

**Comment by dpaquette** [comment]
I have made the changes you recommended

**Comment by stimms** [comment]
Awesome, looks good. I'll find some time in the next little while to pull it into the book and do another release.

**Pull request merged**